README.md

```diff
@@ -1,36 +1,24 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## Introduction
+The purpose of this application is to give the user a graphical user interface for the Radiance/HDRGen pipeline process. The program works by taking in image files and calibration files in order to return light calculated images containing radiance, luminance and color. The application is intended for interior light design researchers who are concerned with how lighting in a room affects visual discomfort.
```

Comment:
For the introduction, you might want to check the original repository's README https://github.com/radiantlab/HDRICalibrationTool has it was vetted by Clotilde already?